### ISSUE TITLE
[Screen Reader-settings-About] Link purpose is not descriptive enough for the all links on about pane

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2589]
**User Experience:**
If a link doesn't have valid text to identify itself, then screen readers will not be able to read the link clearly, it will just be read out to the visually impaired user as “link.” which then makes it difficult for the blind users to understand the purpose or meaning on the link.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Settings control lying in the header section on the page and select it.
4--Navigate to the About menu option listed under the Settings menu control and select it.
5-Navigate to the various controls lying in the pane opened.
6-Verify, All links on about pane is descriptive enough or not.

**Actual Result**
Link purpose is not descriptive enough for the ”0.13.3 and 0.13.4 and v2.0.0-rc11” links on about pane. Due to this narrator is reading only ‘0.13.3’ link.

**Expected Result**
Link Purpose should be descriptive for all '0.13.3 and 0.13.4 and v2.0.0-rc11' links. It should clearly provide the purpose of the link for example ‘Makecode.microbit.org version 0.13.3 link’.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Suggested fix:-**
1-Providing link text that describes the purpose of a link or a link for anchor elements
2-Providing text alternatives for the area elements of image maps
3-Using scripts to change the link text (choose short or long link text )
4- Use 'aria-labelledby' or 'aria-label' for link purpose

**Key References for Fix:**
http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html

**Please refer attachment for more details:**
[MAS2.4.7_About_Link purpose is not descriptive enough for all links.zip](https://github.com/Microsoft/pxt/files/1193752/MAS2.4.7_About_Link.purpose.is.not.descriptive.enough.for.all.links.zip)

**Comment by v-mesala** [comment]
Bug is verified on URL :https://makecode.microbit.org/acc and bug is fixed.Hence closed the bug and added the tag Closed.
